OWNERS

```diff
@@ -18,6 +20,8 @@ filters:
     - mfojtik
     - marun
     - tnozicka
+    - mrunalp
+    - sjenning
 
```

Comment:
Controversial question: will @mrunalp and @sjenning do Kubernetes rebases in the future? I am totally fine with more reviewers here. I am not fine with adding approvers (authority) without having the responsibility for the rebase.

/hold